src/lib/tusktask/context/TaskContext.tsx

```
@@ -34,6 +35,14 @@ export interface TaskContextValues {
   taskDeleteKey: string | null;
   setTaskDeleteKey: SetStateAction<string | null>;
   updateTask: MutateFunction<TaskType | null, TasksPatchRequest>;
+  reScheduleDialog: ReScheduleDialog;
+  setReScheduleDialog: SetStateAction<ReScheduleDialog>;
+  handleResetReScheduleDialog: () => void;
+}
+
+export interface ReScheduleDialog {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'ReScheduleDialog'. Consider to delete it or rename it.

'ReScheduleDialog' is defined here:


(lint/suspicious/noRedeclare)